content/broker/broker-transition/usage/soap/_index.en.md

```diff
@@ -0,0 +1,69 @@
+|                       |[Service Basic](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalBasic.svc)         |                               |
+|                       |[Service EC](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalEC2.svc)            |                               |
+|BrokerService Streamed |[Streamed WS](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamed.svc)      | Handles file data requests.   |
+|                       |[Streamed Basic](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamedBasic.svc) |                               |
+|                       |[Streamed EC](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamedEC2.svc)    |                               |
+|Receipt                |[Receipt WS](https://www.altinn.no/IntermediaryExternal/ReceiptExternal.svc)                     | Handles receipt requests.     |
+|                       |[Receipt Basic](https://www.altinn.no/IntermediaryExternal/ReceiptExternalBasic.svc)                |                               |
+|                       |[Receipt EC](https://www.altinn.no/IntermediaryExternal/ReceiptExternalEC2.svc)                 |                               |
+
+For examples of SOAP implementation see [existing documentation (norwegian only)](https://altinn.github.io/docs/api/soap/grensesnitt/#brokerservice--formidlingstjenester-ws)
+
+### BrokerService Endpoint
+
+The BrokerService endpoint contains functions related to BrokerService metadata requests.
+
+#### InitiateBrokerService
+
+No functional difference for Transitioned services.
+
+#### GetAvailableFiles
+
+No functional difference for Transitioned services. In order to retrieve Altinn 3 files, the service code and the service edition code parameters must be supplied. 
+
+#### ConfirmDownloaded
+
+No functional difference for Transitioned services.
+
+#### CheckIfAvailableFiles
+
+No functional difference for Transitioned services. In order to check for available Altinn 3 files, the service code and the service edition code parameters must be supplied. 
+
+### BrokerServiceStreamed Endpoint
+
+The BrokerServiceStreamed endpoint contains functions related to BrokerService File Data requests.
+
+#### UploadFileStreamed
+
+No functional difference in the call. The reply will contain a pseudo-receipt generated from the Altinn 3 file metadata. ReceiptId will be 0.
+{{% notice warning  %}}
+Since the Upload, and Upload processing of the file is an asynchronous process in Altinn 3, the immediate Receipt status received from the call will not necessarily reflect the final state of the file.
+
+If possible, you should consider adding steps that make calls to [Get File Details](#get-file-details-outbox-sender) to ensure that the file has been successfully processed.
```

Comment:
link invalid, copy-paste from Rest? Consider adding example of good/bad statuses and potentially example request/response.

Reply:
some updates done